### ISSUE TITLE
Undefined index: in Protobuf/Internal/DescriptorPool.php on line 112

### ISSUE THREAD

**Issue by yaronsumel** [google/protobuf#3519]
What's going on with that error ?

```bash
PHP Notice:  Undefined index: Paymentpb\getRequest in ****/vendor/google/protobuf/php/src/Google/Protobuf/Internal/DescriptorPool.php on line 112
PHP Fatal error:  Uncaught Error: Call to a member function getField() on null in ****/vendor/google/protobuf/php/src/Google/Protobuf/Internal/Message.php:79
Stack trace:
#0 ****/src/Paymentpb/getRequest.php(23): Google\Protobuf\Internal\Message->__construct()
#1 ****/index.php(16): Paymentpb\getRequest->__construct()
#2 {main}
  thrown in ****/vendor/google/protobuf/php/src/Google/Protobuf/Internal/Message.php on line 79
```

```php
<?php
require dirname(__FILE__).'/vendor/autoload.php';
$client = new \Paymentpb\PaymentClient('localhost:50051', [
    'credentials' => Grpc\ChannelCredentials::createInsecure(),
]);
$request = new Paymentpb\getRequest();
?>
```

```json
{
    "name": "digota/digota-php",
    "description": "example for PHP",
    "require": {
      "grpc/grpc": "^v1.3.0",
      "google/protobuf": "^v3.4.0"
    },
    "autoload": {
      "psr-4": {
        "": "src/"
      }
    }
  }
```

**Comment by TeBoring** [comment]
Where do you generate code?

**Comment by TeBoring** [comment]
Could you list all your command, proto definition and test code? (Perhaps a github repository to reproduce the bug? Maybe a simplified version without too much detail of your project.)

**Comment by yaronsumel** [comment]
proto
```proto
syntax = "proto3";
package hellopb;
service HelloWorld {
    rpc Hello (helloRequest) returns (helloResponse) {
    }
}
message helloRequest {
}
message helloResponse {
}
```
cmd
```bash
(protoc \
	-I=. \
	-I=../../../ \
	-I=../../gogo/protobuf/protobuf \
	--php_out=_php \
	--grpc_out=_php \
	--plugin=protoc-gen-grpc=/tmp/grpc/bins/opt/grpc_php_plugin \
	hello.proto)
```
index.php
```php
<?php
require dirname(__FILE__).'/vendor/autoload.php';
$client = new \Hellopb\HelloWorldClient('localhost:50051', [
    'credentials' => Grpc\ChannelCredentials::createInsecure(),
]);
$request = new \Hellopb\helloRequest();
?>
```

**Comment by yaronsumel** [comment]
same error for 
```bash
php -d extension=grpc.so index.php
````

**Comment by TeBoring** [comment]
I think the work around is to rename helloRequest and helloResponse to HelloRequest and HelloResponse

**Comment by TeBoring** [comment]
Fixed in #3537

**Comment by yaronsumel** [comment]
Thanks.
